drivers/wifi/nrf700x/src/wifi_mgmt.c

```diff
@@ -428,6 +429,10 @@ int nrf_wifi_twt_teardown_flows(struct nrf_wifi_vif_ctx_zep *vif_ctx_zep,
 		}
 		/* UMAC doesn't send TWT teardown event for host initiated teardown */
 		nrf_wifi_twt_update_internal_state(vif_ctx_zep, false, flow_id);
+		/* Raise a tear down event from host itself, till UMAC supports */
```

Comment:
Don't we need to encode the status i.e. `SUCCESS` or `FAIL` ?

Reply:
For host genearted, it will be always success. There is no command status field. Only for peer response of sucess/failure. 
Do I need to add for a command status field as this is temporary till UMAC gives proper event?

Reply:
Yeah, I think it's better to keep success/failure. I will update accordingly

Reply:
```suggestion
		/* TODO: Remove this once UMAC sends the status */
```

Reply:
Yes, we should always inform the status

Reply:
Yeah, I will update.